jaxb-adapters/src/main/java/org/hawkular/accounts/jaxb/adapters/ZonedDateTimeXmlAdapter.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.accounts.jaxb.adapters;
+
+import javax.xml.bind.annotation.adapters.XmlAdapter;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * @author Juraci Paixão Kröhling
+ */
+public class ZonedDateTimeXmlAdapter extends XmlAdapter<String, ZonedDateTime> {
+    private final DateTimeFormatter formatter = DateTimeFormatter.ISO_DATE_TIME;
+
+    @Override
+    public ZonedDateTime unmarshal(String s) throws Exception {
+        if (null == s || s.isEmpty()) {
+            return null;
+        }
+
+        return ZonedDateTime.parse(s, formatter);
+    }
+
+    @Override
+    public String marshal(ZonedDateTime zonedDateTime) throws Exception {
+        if (null == zonedDateTime) {
+            return null;
+        }
+
+        return zonedDateTime.withZoneSameInstant(ZoneOffset.UTC).format(formatter);
```

Comment:
Yes, I agree that the TZ of the server makes no sense. UTC is much better.

If I was developing a client software, I'd perhaps like a possibility to tell the server which TZ I want to get the dates in. I guess we do not have anything like that?